modules/sc-mesh-secure-deployment/configure.sh

```diff
@@ -105,19 +144,18 @@ function server {
 
 function client {
   echo '> Configuring the client...'
-  # Make the client
+  # Make the server
   pushd .
   cd ../..
-  make mesh_tb_client
+  make client
   popd
   # Connect to the same AP as the server
-  read -p "> We MUST be on the same network as the Server. Connect to AP? (Y/N): " confirm
+  read -p "> We need to be connect to the same network as the server... Connect to an Access Point? (Y/N): " confirm
```

Comment:
as a suggestion: we should see if the interface is already connected to an access point with `iwconfig ` to avoid asking again to reconnect

Reply:
Good idea, will make this change and push.

Reply:
Ah we need to actually figure out a better way of doing this as the config is needed for the 1st node joined (i.e. gateway). Will create an issue.